.travis.yml

```diff
@@ -14,8 +14,7 @@
 
 language: go
 go:
-- 1.16
-
+- "1.16"
```

Comment:
should it be in quotes?

Reply:
Yes! I found in Travis documentation. It is specially important when referring to a 0-ended version. The example was that, for Go 1.10, if you don't use quotes, you will get the 1.1 version instead of the desired 1.10, because it is considered as a float number :-)